src/pages/dashboard/dashboardState.js

```diff
@@ -0,0 +1,344 @@
+import {
```

Comment:
Do we need this?

Reply:
> Do we need this?

Please check JS pageSpecificStatsController.js:

Reply:
pageSpecificStatsController.js:

 import { getFilteredStat, fetchDataForDashboardDataTable } from '../../pages/dashboard/dashboardState';

Reply:
Can you update this to use dashboardreducer.js  so we don't use this entire file